streams-tests/jvm/src/test/scala/zio/stream/StreamChunkSpec.scala

```diff
@@ -14,6 +14,18 @@ object StreamChunkSpec extends ZIOBaseSpec {
 
   import ZIOTag._
 
+  def tinyChunks[R <: Random, A](a: Gen[R, A]): Gen[R with Sized, Chunk[A]] =
+    Gen.chunkOfBounded(0, 3)(a)
+
+  def smallChunks[R <: Random, A](a: Gen[R, A]): Gen[R with Sized, Chunk[A]] =
+    Gen.small(Gen.chunkOfBounded(0, _)(a))
```

Comment:
If you're just bounding it at 0 you can just do `Gen.small(Gen.chunkOfN(_)(a))`.